clusterman/aws/auto_scaling_resource_group.py

```diff
@@ -246,7 +248,7 @@ def _get_options_for_instance_type(
             weight = weight or self.market_weight(instance_market)
             options.append(
                 ClusterNodeMetadata(
-                    agent=AgentMetadata(total_resources=ClustermanResources.from_instance_type(instance_type)),
+                    agent=AgentMetadata(total_resources=ClustermanResources.from_instance_type(instance_type, cluster)),
```

Comment:
No need to have cluster passed as a function argument, you can read it as a property from `self` directly.